toolkit/scripts/containerized-build/create_container_build.sh

```diff
@@ -228,25 +228,25 @@ cp resources/setup_functions.sh $tmp_dir/setup_functions.sh
 sed -i "s~<TOPDIR>~${topdir}~" $tmp_dir/setup_functions.sh
 # TODO: Remove when PMC is available for 3.0
 if [[ "${version}" == "3.0" ]]; then # Add 3.0 DailyBuild repo
-    cp resources/mariner-3_repo $tmp_dir/mariner-3_repo
-    sed -i "s~<DAILY_BUILD_ID>~${DAILY_BUILD_ID}~" $tmp_dir/mariner-3_repo
+    cp resources/azl-3_repo $tmp_dir/azl-3_repo
+    sed -i "s~<DAILY_BUILD_ID>~${DAILY_BUILD_ID}~" $tmp_dir/azl-3_repo
     if [[ $(uname -m) == "x86_64" ]]; then
-        sed -i "s~<ARCH>~x86-64~" $tmp_dir/mariner-3_repo
+        sed -i "s~<ARCH>~x86-64~" $tmp_dir/azl-3_repo
     else
-        sed -i "s~<ARCH>~aarch64~" $tmp_dir/mariner-3_repo
+        sed -i "s~<ARCH>~aarch64~" $tmp_dir/azl-3_repo
     fi
 fi
 
 # ============ Build the image ============
-dockerfile="${script_dir}/resources/mariner.Dockerfile"
+dockerfile="${script_dir}/resources/azl.Dockerfile"
 
 # TODO: Remove test mode when image is available for 3.0
 if [[ "${mode}" == "build" || "${mode}" == "test" ]]; then # Configure base image
     echo "Importing chroot into docker..."
     chroot_file="$BUILD_DIR/worker/worker_chroot.tar.gz"
     if [[ ! -f "${chroot_file}" ]]; then build_worker_chroot; fi
     chroot_hash=$(sha256sum "${chroot_file}" | cut -d' ' -f1)
-    container_img="mcr.microsoft.com/cbl-mariner/containerized-rpmbuild:${version}"
+    container_img="azl/containerized-rpmbuild:${version}"
```

Comment:
Are we sure this is right? I'll believe you if it is, but I'm surprised it isn't `mcr.microsoft.com/azl`.

Reply:
I'm not sure either. If we are tagging images locally, with no intention of publishing them, does prefix `mcr.microsoft.com` matter? What is the protocol to follow?
We still download from `mcr.microsoft.com/cbl-mariner/..` if not building the image locally.

Reply:
Based on this: https://github.com/microsoft/azurelinux/blob/28d18aeeb900f1eba849921141d817fdf1e57918/.pipelines/containerSourceData/scripts/BuildGoldenContainer.sh#L51C1-L56C64, I think we may want something like:
```
mcr.microsoft.com/azurelinux/containerized-rpmbuild:3.0
```
But verify.

Reply:
Cool, let's do that so we have consistency